torch_ort/docker/Dockerfile.ort-default-stable-torch1.12.1-onnxruntime190-cu116-cudnn8-devel-ubuntu18.04

```diff
@@ -3,39 +3,43 @@
 # Licensed under the MIT License.
 
 # CUDA development image for building sources
-FROM nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04 as builder
-
-ARG TORCH_CUDA_VERSION=cu102
-ARG TORCH_VERSION=1.8.1
-ARG TORCHVISION_VERSION=0.9.1
+FROM nvidia/cuda:11.6.2-cudnn8-devel-ubuntu18.04 as builder
+ARG TORCH_CUDA_VERSION=cu116
+ARG ONNXRUNTIME_TRAINING_VERSION=1.12.0
+ARG TORCH_VERSION=1.12.1
+ARG TORCHVISION_VERSION=0.13.1
 
 # Install and update tools to minimize security vulnerabilities
 RUN apt-get update
 RUN apt-get install -y software-properties-common wget apt-utils patchelf git libprotobuf-dev protobuf-compiler cmake
 RUN unattended-upgrade
 RUN apt-get autoremove -y
 
+# install miniconda (comes with python 3.9 default)
+RUN apt-get install curl
+ARG CONDA_VERSION=py39_4.10.3
+ARG CONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh
+RUN curl -fSsL --insecure ${CONDA_URL} -o install-conda.sh &&\
+    /bin/bash ./install-conda.sh -b -p /opt/conda &&\
+    /opt/conda/bin/conda clean -ya
+ENV PATH=/opt/conda/bin:${PATH}
+
 # Python and pip
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 RUN apt-get install -y python3-pip
 RUN update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 RUN pip install --upgrade pip
 
 # PyTorch
-# TODO: upgrade to latest onnx
-RUN pip install onnx==1.11.0 ninja
-RUN pip install torch==${TORCH_VERSION}+${TORCH_CUDA_VERSION} torchvision==${TORCHVISION_VERSION}+${TORCH_CUDA_VERSION} -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip install onnx ninja
+RUN pip install torch==${TORCH_VERSION}+${TORCH_CUDA_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/torch/
 
 # ORT Module
-RUN pip install --pre onnxruntime-training -f https://download.onnxruntime.ai/onnxruntime_nightly_cu102.html
+RUN pip install --pre onnxruntime-training -f https://download.onnxruntime.ai/onnxruntime_nightly_cu116.html
```

Comment:
Let's change this to the stable release of `onnxruntime-training` 1.13.0. That way the pipeline is ready for the next release.